pathd/path_pcep_cli.c

```diff
@@ -2043,7 +2043,7 @@ DEFPY(pcep_cli_no_pcc,
 
 DEFPY(pcep_cli_pcc_pcc_msd,
       pcep_cli_pcc_pcc_msd_cmd,
-      "[no] msd (1-32)",
+      "[no] msd [(1-32)]",
```

Comment:
if I enter just `msd` is that handled correctly down the path?

Reply:
it is ignoring the command.
So basically, if you have 4, you will keep 4.
If you have 6, you will keep 6.